mk/spksrc.python-wheel.mk

```diff
@@ -25,7 +25,10 @@ PYTHONPATH = $(PYTHON_LIB_NATIVE):$(INSTALL_DIR)$(INSTALL_PREFIX)/$(PYTHON_LIB_D
 
 ### Python wheel rules
 build_python_wheel:
-	@$(RUN) PYTHONPATH=$(PYTHONPATH) $(HOSTPYTHON) -c "import setuptools;__file__='setup.py';exec(compile(open(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" $(BUILD_ARGS) bdist_wheel -b $(WORK_DIR)/wheelbuild -d $(WORK_DIR)/wheelhouse
+	@$(RUN) PYTHONPATH=$(PYTHONPATH) $(HOSTPYTHON) -c "import setuptools;\
```

Comment:
I could have kept the one-liner but I thought it was becoming hard to read.

Reply:
There is another (open) option which would be to patch the native python so that an environment variable (`TARGET_EXT_SUFFIX` ?) could override the `EXT_SUFFIX` variable computed upon compiling.

Reply:
@smaarn I find it strange there is no standard "out-of-the-box" cross-compilation support for wheels with Python 3.

Reply:
@ymartin59 I found this `crossenv`(https://pypi.org/project/crossenv/) python module which seems to be targetting cross compilation, the only thing is that it could mean some more changes to be made. From what I gathered it meant creating a shell script which would appropriately alias `python` to target a fake python which would be emulating the environment information for the target architecture.

Reply:
@smaarn Correct. I have already been told about it. I will give another try to cross-compile and run borgbackup on current Python 3.6.10